src/components/MyRecipes.vue

```diff
@@ -1,291 +1,540 @@
-                            <div class="meta">
-                                <span>
-                                    {{ recipe.summary | truncate(115, '...') }}
-                                </span>
-                            </div>
-                        </div>
-                    </div>
-                </div>
+            <div class="content">
+              <a class="header" href="">
+                <small>
+                  <span>
+                    <router-link :to="{ name: 'Recipe', params: { slug: recipe.slug } }">
+                      {{ recipe.name }}
+                    </router-link>
+                  </span>
+                </small>
+              </a>
+              <span style="float: right !important; font-size: 16px">
+                <router-link :to="{ name: 'EditRecipe', params: { slug: recipe.slug } }">
+                  edit
+                </router-link>
+              </span>
+              <div class="meta">
+                <span>
+                  {{ recipe.summary | truncate(115, "...") }}
+                </span>
+              </div>
             </div>
+          </div>
         </div>
+      </div>
     </div>
+  </div>
 </template>
 
 <script>
-import UploadImage from './UploadImage.vue';
+import UploadImage from "./UploadImage.vue";
 import { VueEditor } from "vue2-editor";
-import FlagPicker from './Widgets/FlagPickerWidget.vue'
+import FlagPicker from './Widgets/FlagPickerWidget.vue';
+import { ValidationProvider, ValidationObserver } from 'vee-validate';
+import { extend } from 'vee-validate';
+import * as rules from 'vee-validate/dist/rules';
+import { messages } from 'vee-validate/dist/locale/en.json';
+import CookbookSelector from './Widgets/CookbookSelectorWidget';
+import { mapActions } from 'vuex';
+import { debounce } from 'lodash';
+import { createClient } from 'pexels';
+
 
 export default {
-    name: "MyRecipes",
-    mounted() {
-        let username = this.$store.state.username
-        this.$store.dispatch('fetch_contributor', username)
-        this.$store.dispatch('reset_msgs')
+  name: "MyRecipes",
+  mounted() {
+    let username = this.$store.state.username
+    this.$store.dispatch('fetch_contributor', username)
+    this.$store.dispatch('reset_msgs')
+   
+  },
+
+  created() {
+    Object.keys(rules).forEach(rule => {
+      extend(rule, {
+        ...rules[rule], // copies rule configuration
+        message: messages[rule] // assign message
+      });
+    })
+  },
+
+  computed: {
+    _categories() {
+      let cs = this.$store.state.cookbookStore.definitions.categories.contents
+      return JSON.parse(cs)
     },
-    computed: {
-        _categories() {
-            let cs = this.$store.state.cookbookStore.definitions.categories.contents
-            return JSON.parse(cs)
-        },
-        _myRecipes() {
-            let recipes = this.$store.state.contributor.recipes
-            return recipes.filter(function(recipe) {
```

Comment:
Hmm maybe update this to filter out drafts. Is there a reason why this was changed?